pkg/core/codeset_store.go

```diff
@@ -8,16 +8,21 @@ import (
 	"github.com/fuseml/fuseml-core/pkg/domain"
 )
 
+type codesetID struct {
+	codesetName    string
+	codesetProject string
```

Comment:
As `codesetID` is meant to describe codeset, I think you should just just `Name` and `Project`

Reply:
good point